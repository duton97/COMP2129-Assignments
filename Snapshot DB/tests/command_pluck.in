SET a 1 2 3
GET a
PLUCK a 2
GET a
PLUCK a 0
PLUCK b 1
BYE
