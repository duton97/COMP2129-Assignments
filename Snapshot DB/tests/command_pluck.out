> ok

> [1 2 3]

> 2

> [1 3]

> index out of range

> no such key

> bye
